tap-version-14-specification.md

```diff
@@ -0,0 +1,741 @@
+
+```
+TAP version 14
+1..0 # skip English-to-French translator isn't installed
+```
+
+## Got spare tuits?
+
+The following example reports that four tests are run and the last two
+tests failed. However, because the failing tests are marked as things
+to do later, they are considered successes. Thus, a harness should
+report this entire listing as a success.
+
+```
+TAP version 14
+1..1
+    # Subtest: child program
+    1..4
+    ok 1 - Creating test program
+    ok 2 - Test program runs, no error
+    not ok 3 - infinite loop # TODO halting problem unsolved
+    not ok 4 - infinite loop 2 # TODO halting problem unsolved
+ok 1 - child program
+```
+
+## Creative liberties
+
+This listing shows an alternate output where the test numbers aren't
+provided. The test also reports the state of a ficticious board game
+as a YAML block. Finally, the test count is reported at the end.
+
+```
+TAP version 14
+ok - created Board
+ok
+ok
+ok
+ok
+ok
+ok
+ok
+  ---
+  message: "Board layout"
+  severity: comment
+  dump:
+     board:
+       - '      16G         05C        '
+       - '      G N C       C C G      '
+       - '        G           C  +     '
+       - '10C   01G         03C        '
+       - 'R N G G A G       C C C      '
+       - '  R     G           C  +     '
+       - '      01G   17C   00C        '
+       - '      G A G G N R R N R      '
+       - '        G     R     G        '
+  ...
+ok - board has 7 tiles + starter tile
+1..9
+```
+
+## CHANGES FROM TAP13
+
+- YAML blocks standardized to 2 space indentation
+- Subtests specified to behavior of `Test::More` and `node-tap`.
+- Normative advice regarding exit code for harness programs
+- Examples and usage comments made language-agnostic.
+- Clarification of whitespace and hyphens in test lines.
+- Clarification of handling of incorrect lines.
+- Specification of Pragma lines
+
+## BUGS
+
+Please report issues with this specification to the
+[TestAnything/Specification
+project](https://github.com/TestAnything/Specification).
+
+## AUTHORS
+
+The original TAP documentation (of which this is a hacked about
+version) was written by Andy Lester, based on the original
+`Test::Harness` documentation by Michael Schwern.
+
+TAP13 documentation written by Andy Armstrong.
+
+TAP14 documentation written by Isaac Z. Schlueter.
+
+## ACKNOWLEDGEMENTS
+
+Thanks to Pete Krawczyk, Paul Johnson, Ian Langworth and Nik Clayton
+for help and contributions on this document.  The basis for the TAP
+format was created by Larry Wall in the original test script for Perl
+1. Tim Bunce and Andreas Koenig developed it further with their
+modifications to `Test::Harness`.
+
+As of this writing, the TAP specification is now managed by the
+[TestAnything organization](https://github.com/TestAnything).
+
+## COPYRIGHT
+
+Copyright Michael G Schwern <schwern@pobox.com>, Andy Lester
+<andy@petdance.com>, Andy Armstrong <andy@hexten.net>, Isaac Z.
+Schlueter <i@izs.me>.
```

Comment:
I added myself because writing TAP14 is a substantial shift from TAP13.  There are a lot of new words I've put here, and I'm not comfortable assigning responsibility for those to others.